src/hostnet/utils.mli

```diff
@@ -7,3 +7,9 @@ val rtlGenRandom: int -> bytes option
 
 val setSocketTTL: Unix.file_descr -> int -> unit
 (** [setSocketTTL s ttl] sets the TTL on the socket [s] to [ttl] *)
+
+val cstruct_send: Unix.file_descr -> Cstruct.t -> int
```

Comment:
I really would prefer to get these stubs into Cstruct_unix. It seems safe enough to put these alongside `Cstruct_unix.of_fd`

Reply:
yeah makes sense, I'm about to write `writev` and `readv` too

Reply:
https://github.com/mirage/ocaml-cstruct/pull/302